static/app/views/organizationGroupDetails/eventCreatedTooltip.tsx

```diff
@@ -55,10 +61,20 @@ export default function EventCreatedTooltip({event}: Props) {
             <br />
             {dateReceived.format(format)}
           </dd>
-          <dt>{t('Latency')}</dt>
-          <dd>{formatDateDelta(dateCreated, dateReceived)}</dd>
         </Fragment>
       )}
+      {
+        <Fragment>
+          <dt>{t('Latency')}</dt>
+          <dd>
+            {dateCreated && dateReceived ? (
+              formatDateDelta(dateCreated, dateReceived)
+            ) : (
+              <NotApplicableText>{t('n/a')}</NotApplicableText>
+            )}
+          </dd>
```

Comment:
Couldn't this be nested at line 59 or does doing that do something weird with styling?

Reply:
cleaned it up!